tests/test_run.rs

```diff
@@ -256,7 +438,12 @@ fn test_run_e2e(
         proof: tmp_dir.path().join("proof.json"),
     };
 
-    match run_cairo1(&prove_args, &tmp_dir) {
+    let run_result = match cairo_version {
+        CairoVersion::cairo0 => run_cairo0(&prove_args, &tmp_dir).map_err(|e| anyhow::anyhow!(e)),
+        CairoVersion::cairo1 => run_cairo1(&prove_args, &tmp_dir),
+    };
+
+    match run_result {
         Ok(_) => {
             let filename = program_file.file_stem().unwrap().to_str().unwrap();
             let air_public_input = tmp_dir
```

Comment:
There is a lot of nested error handling around here.
Consider just doing result.expect("XYZ went wrong") throughout.

Reply:
Done